relay-server/src/utils/sizes.rs

```diff
@@ -32,9 +35,18 @@ pub fn check_envelope_size_limits(
     let mut event_size = 0;
     let mut attachments_size = 0;
     let mut session_count = 0;
+    let mut span_count = 0;
+    let mut log_count = 0;
     let mut client_reports_size = 0;
 
     for item in envelope.items() {
+        if item.is_container() && item.len() > config.max_container_size() {
+            return Err(item
+                .attachment_type()
```

Comment:
Why use the attachment type here?

Reply:
Oh shoot, you're right. It's a container. Will fix.